lib/smart_proxy_container_gateway/container_gateway_api.rb

```diff
@@ -164,12 +175,15 @@ class Api < ::Sinatra::Base
           # 'expires_in' is an optional field. If not provided, assume 60 seconds per OAuth2 spec
           expires_in = token_response_body.fetch("expires_in", 60)
           expires_at = token_issue_time + expires_in.seconds
-
-          container_gateway_main.insert_token(
-            request.params['account'],
-            token_response_body['token'],
-            expires_at.rfc3339
-          )
+          if request.params['account'].present?
+            container_gateway_main.insert_token(
+              request.params['account'],
+              token_response_body['token'],
+              expires_at.rfc3339
```

Comment:
In case request.params['account'] is unavailable which is the case when flatpak isn't using authentication aka user is logged out and oci_auth.json is removed, return unauthorized instead of ttrying to insert empty username in token.

Reply:
Huh..Will update.. present? isn't ruby..It's rails.. 😭